src/pages/projectDashboard/sections/ProjectWallet.tsx

```diff
@@ -114,6 +116,7 @@ export const ProjectWallet = () => {
           lndConnectionDetailsInput: createWalletInput?.lndConnectionDetailsInput,
           lightningAddressConnectionDetailsInput: createWalletInput?.lightningAddressConnectionDetailsInput,
           id: projectWallet?.id,
+          feePercentage,
```

Comment:
In this case, I think it would be better to use it from the createWalletInput, as I think only have the feePercentage for nodes, not for lightning, and we already have the logic placed inside createWalletInput to separate the payload for lighting and nodes, in the useWalletForm for this.